essentials-framework/Essentials Core/PepperDashEssentialsBase/Feedbacks/BoolFeedback.cs

```diff
@@ -18,6 +18,7 @@ public class BoolFeedback : Feedback
         /// </summary>
         public override bool BoolValue { get { return _BoolValue; } }
         bool _BoolValue;
+		public bool AlwaysSendValue;
```

Comment:
If this is a public value, it should be a property with a private setter. Also, it should be moved to the `FeedbackBase` class to prevent duplication of code, along with a similar constructor there.